service/project.go

```diff
@@ -134,9 +152,10 @@ func (s *ProjectService) CreateEnvironment(ctx context.Context, c model.Environm
 func (s *ProjectService) GetEnvironment(ctx context.Context, projectID, envID, authUserID uuid.UUID) (model.Environment, error) {
```

Comment:
Since we are here, another thing that constantly irritates my eye, albeit only slightly, is - why do we return zero struct when the environment does not exist?
Wouldn't it be simpler to have a pointer there?
It would also allow for simpler checks in the transport layer, if it decides (yes, we return `not exists error` also, but often you just want to be able to do simple `if env == nil ...`.

I personally in my experience never had problems with pointer approach.

Applicable to the rest of domains obviously.

Reply:
Also probably a separate PR (if you agree)

Reply:
Created issue for it #24